tests/counters.rs

```diff
@@ -71,6 +72,27 @@ mod counters {
         assert_eq!(counter_val.to_string(), resp.bytes().await.unwrap());
     }
 
+    #[tokio::test]
+    #[serial]
+    async fn get_config() {
+        cleanup().await;
+
+        let counter_name = "test_counter";
+        let initial_val = 10;
+
+        let _ = run(&counters::create_strong(counter_name).with_value(initial_val)).await;
+        let resp = run(&counters::get_config(counter_name)).await;
+
+        assert!(resp.status().is_success());
+
+        let config_bytes = resp.bytes().await.unwrap();
+        let config_str = std::str::from_utf8(&config_bytes).unwrap();
```

Comment:
I think you can use here `str::from_utf8` without `std::` since it should be `use`d implicitly via the prelude.

Reply:
No, I tried it and it doesn't work.

Reply:
Then use `core::str` for maximum future-proofing.